<launch>
	<node pkg="final_race" type="rainbow_road.py" name="rainbow_road" output="screen">
		<param name="pub_topic" value="/odom"/>
        <param name="sub_topic" value="/odom"/>
	</node>
</launch>
